pkg/networkutils/network.go

```diff
@@ -451,23 +453,22 @@ func (n *linuxNetwork) buildIptablesSNATRules(vpcCIDRs []string, primaryAddr *ne
 			"-m", "comment", "--comment", "AWS SNAT CHAIN", "-j", "AWS-SNAT-CHAIN-0",
 		}})
 
-	for i, cidr := range allCIDRs {
-		curChain := chains[i]
-		curName := fmt.Sprintf("[%d] AWS-SNAT-CHAIN", i)
-		nextChain := chains[i+1]
+	for _, cidr := range allCIDRs {
+		curChain := "AWS-SNAT-CHAIN-0"
+		curName := "AWS-SNAT-CHAIN-0"
```

Comment:
nit: these should be defined outside the for loop since they are the same for each iteration